PklGenerator/FileSource.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.CodeAnalysis.Text;
+
+namespace PklGenerator;
+
+internal class FileSource : IParserSource
+{
+    private string FilePath { get; set; }
```

Comment:
NIT / question: is there any reason why we'd prefer an autoproperty for this private member instead of just a regular private field?